web-client/src/presenter/computeds/formattedCaseDetail.js

```diff
@@ -263,9 +268,15 @@ export const formattedCaseDetail = (get, applicationContext) => {
     d => d.isOnDocketRecord,
   );
 
-  result.formattedPendingDocketEntriesOnDocketRecord = result.formattedDocketEntriesOnDocketRecord.filter(
-    d => d.pending && d.servedAt,
-  );
+  if (isCodeEnabled(7198)) {
+    result.formattedPendingDocketEntriesOnDocketRecord = result.formattedDocketEntriesOnDocketRecord.filter(
+      d => d.pending && (d.servedAt || d.isLegacyServed),
```

Comment:
Here's another spot where `pending && (servedAt || isLegacyServed)` is used.

Reply:
Refactoring card added https://trello.com/c/BQLzXAuD